framework/src/linearfvbcs/LinearFVBoundaryCondition.C

```diff
@@ -49,7 +49,7 @@ LinearFVBoundaryCondition::LinearFVBoundaryCondition(const InputParameters & par
     MooseVariableInterface(this,
                            false,
                            "variable",
-                           Moose::VarKindType::VAR_SOLVER,
+                           Moose::VarKindType::VAR_ANY,
```

Comment:
why this change?

Reply:
this is allowing BCs on aux variables. Peter will do a separate PR

Reply:
Well, this is in this PR:
https://github.com/idaholab/moose/pull/29532